### ISSUE TITLE
Statoscope as webpack plugin

### ISSUE THREAD

**Issue by shanmukh15** [smelukov/statoscope#28]
Hi, 

I am trying out statoscope@ 3.1.0 for one of my project on webpack 5. When i tried to add the two import statements and added the init to plugins of webpack config. During webpack build, it keeps throwing me window is not defined.

**Comment by smelukov** [comment]
Hi! Statoscope does not have webpack plugin for now. It's just a UI to display stats.
To init the UI, you may create a JS file with the content:
```js
import init from '@statoscope/ui-webpack';
import stats from 'path/to/stats.json'

init({
  name: "stats.json",
  data: stats
});
```
And insert it into any HTML page.

I think I'll add webpack plugin in the next release 🤔

**Comment by smelukov** [comment]
Solved in [3.2.0](https://github.com/smelukov/statoscope/blob/master/packages/ui-webpack/CHANGELOG.md#320-11-december-2020)
